labcodes/labstorybook
# DSYS-18/alerts-js

**Link to task:**
https://labcodes.atlassian.net/browse/DSYS-18

**How to test:**
- With the LabSystem running in `localhost`, go to the _Component/System Messages_ div the main page, then check if everything is rendered as expected.
- Locally, from `labstorybook` dir, run npm test, then check if all tests are passing.

**Description of your solution:**
This PR implements the JS behavior of Alerts and Banners (scroll handler, button response etc.). It also introduces a `SystemMessagesPlayground` component to the documentation on `SystemMessages.stories`. Small fixes to Stories and CSS are included. Snapshots are updated.
